static/js/components/ProfileProgressControls.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Button from 'react-mdl/lib/Button';
+
+import { saveProfileStep } from '../util/profile_edit';
+
+export default class ProfileProgressControls extends React.Component {
+  static propTypes = {
+    nextUrl: React.PropTypes.string,
+    prevUrl: React.PropTypes.string,
+    isLastTab: React.PropTypes.bool,
+    saveProfile: React.PropTypes.func.isRequired,
```

Comment:
It's used by `saveProfileStep` which pulls `profile` and `saveProfile` from `this.props`